telos-frontend/src/components/calendar/WidgetCalendar.jsx

```diff
@@ -1,13 +1,17 @@
+/* eslint-disable no-unused-vars */
```

Comment:
eslint disables are generally bad practice - what's your reasoning for including them here?

Reply:
Good catch! I actually added them before while testing and forgot to remove. I'll remove it now.
There is one eslint disable in the page component which i have intentionally added. There are some props which we might use soon but are not using currently .

Reply:
They look like they're not necessary here if you get rid of the useState import

Reply:
Sweet yeah the other disable in Page looks fine as long as you give an explanation in the code